antora/supplemental-ui/partials/head-meta.hbs

```diff
@@ -1,8 +1,10 @@
-<!--- https://realfavicongenerator.net/ --->
     <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
     <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
     <link rel="manifest" href="/site.webmanifest">
     <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
     <meta name="msapplication-TileColor" content="#da532c">
     <meta name="theme-color" content="#ffffff">
```

Comment:
All this we want in the website header as well. (Though it could be something done in a future PR.)

Reply:
Should be fixed in b0428831eb83d9abc82ec1fea1d4dde9e2155487, with some duplication